scripts/cli/tasks/cherrypick.ts

```diff
@@ -31,15 +31,18 @@ const cherryPickRunner: TaskRunner<CherryPickOptions> = async () => {
   console.log('Printing PRs with cherry-pick-needed, in ASC merge date order');
   console.log('--------------------------------------------------------------------');
 
-  for (const item of res.data) {
+  for (const index in res.data) {
+    const item = res.data[index];
+    const n = +index + 1;
+
     if (!item.milestone) {
       console.log(item.number + ' missing milestone!');
       continue;
     }
 
     const issueDetails = await client.get(item.pull_request.url);
     console.log(`* ${item.title}, (#${item.number}), merge-sha: ${issueDetails.data.merge_commit_sha}`);
-    commands += `git cherry-pick -x ${issueDetails.data.merge_commit_sha}\n`;
+    commands += `${n}. git cherry-pick -x ${issueDetails.data.merge_commit_sha}\n`;
```

Comment:
the numbers make it a lot harder to just copy paste all the commands

Reply:
That's a very good point. Perhaps I'm a bit too cautious and chose not to copy paste them all at once.

My thinking was that in case of a conflict, you'd want to stop and solve that first. There's no harm done in attempting to cherry-pick in the middle of a cherry-pick conflict since git will scream at you.

I'm happy to remove that change if others find it detrimental to the process.

Reply:
if you copy paste a bunch of commands and one fails it wont execute the rest (in my experiance) and being able to copy paste 10+ at once has been very powerful

Reply:
Makes perfect sense, I have removed it.